internal/cri/server/images/image_remove.go

```diff
@@ -34,17 +34,24 @@ import (
 // Remove the whole image no matter the it's image id or reference. This is the
 // semantic defined in CRI now.
 func (c *GRPCCRIImageService) RemoveImage(ctx context.Context, r *runtime.RemoveImageRequest) (*runtime.RemoveImageResponse, error) {
+	resp, err := c.CRIImageService.RemoveImage(ctx, r.Image)
+	if err != nil {
+		return nil, err
+	}
+	return resp, nil
+}
+
+func (c *CRIImageService) RemoveImage(ctx context.Context, imageSpec *runtime.ImageSpec) (*runtime.RemoveImageResponse, error) {
```

Comment:
maybe a comment line explaining ctx for trace here... and bring over the context api def.. for return values.. or just mention it must meet the CRI api definition...

Reply:
Hmm not sure I understand this. Are you suggesting adding a comment for trace?

Reply:
had a chat with Mike on slack. Documentation for this function has been moved above to the grpc handler function.